schema_registry/client/client.py

```diff
@@ -170,24 +174,20 @@ def _cache_schema(
             if version:
                 self._add_to_cache(self.subject_to_schema_versions, subject, schema, version)
 
-    @property
-    def session(self):  # type: ignore
-        if self._session.is_closed:
-            self._session = self._create_session()
-        return self._session
-
-    @abc.abstractmethod
-    def _create_session(self):  # type: ignore
-        ...
-
+    def request(
```

Comment:
I think this should be an abstract method, right? Child classes must implement this methods.

Reply:
Yes, you're right! Good catch.